### PR TITLE
Support native `Headers` in `verifyRequest`

### PR DESCRIPTION
**Pull request by CTNicholas**
It'd be nice if the native `Headers` object was supported in [`WebhookHandler.verifyRequest`](https://liveblocks.io/docs/api-reference/liveblocks-node#verifyRequest), then it'd work with [Next.js](https://nextjs.org/docs/app/building-your-application/routing/route-handlers#headers) `/app` and other modern frameworks.

This would enable both of these:
```ts
export function POST(request: Request) {
  const event = webhookHandler.verifyRequest({
    headers: request.headers,
    rawBody: "",
  });

  // ...
}
```

```ts
import { headers } from "next/headers";

export function ServerComponent() {
  const event = webhookHandler.verifyRequest({
    headers: headers(),
    rawBody: "",
  });

  // ...
}
``

### CODE REVIEW

File: packages/liveblocks-node/src/webhooks.ts [hunk 1]
```diff
@@ -58,11 +59,18 @@ export class WebhookHandler {
...
    * Verifies the headers and returns the webhookId, timestamp and rawSignatures
    */
-  private verifyHeaders(headers: IncomingHttpHeaders) {
+  private verifyHeaders(headers: IncomingHttpHeaders | Headers) {
     const sanitizedHeaders: IncomingHttpHeaders = {};
-    Object.keys(headers).forEach((key) => {
-      sanitizedHeaders[key.toLowerCase()] = headers[key];
-    });
+
+    if (headers instanceof Headers) {
```
Comment (GuillaumeSalles) [inline packages/liveblocks-node/src/webhooks.ts]: Not sure `instanceof Headers` works in a node environment. I don't think `Headers` is globally available in node. We should probably add an additional check.

Comment (CTNicholas) [reply packages/liveblocks-node/src/webhooks.ts]: Good shout, `Headers` have only been supported in Node.js since `18.0.0`. Added a check.

Comment (FlowFlorent) [reply packages/liveblocks-node/src/webhooks.ts]: Nice, also feel free to add an example with `Headers` in the `WebhookRequest` type if you think it helps!

**Review by FlowFlorent — APPROVED** [review]
one suggestion, otherwise looks good! 🚀

File: packages/liveblocks-node/src/webhooks.ts (hunk 1 above)
Comment (FlowFlorent) [reply packages/liveblocks-node/src/webhooks.ts]: @GuillaumeSalles they do but on the off chance, I'd rather be safe

Comment (CTNicholas) [reply packages/liveblocks-node/src/webhooks.ts]: Done @FlowFlorent!

**Pull request merged**